legal/non-go-dependencies.json

```
@@ -1,7 +1,9 @@
-{
-	"Dependency": "zyantific/zydis",
-	"Version": "v3.1.0",
-	"Licence": "MIT",
-	"URL": "https://zydis.re",
-	"LicenceFile": "https://raw.githubusercontent.com/zyantific/zydis/v3.1.0/LICENSE"
-}
+[
+	{
+		"Dependency": "zyantific/zydis",
+		"Version": "v3.1.0",
+		"Licence": "MIT",
+		"URL": "https://zydis.re",
+		"LicenceFile": "https://raw.githubusercontent.com/zyantific/zydis/v3.1.0/LICENSE"
+	}
+]
```

Not actually your mistake (I forgot to update this), but can we fix this while we are changing this anyway?

```suggestion
[
	{
		"Dependency": "zyantific/zydis",
		"Version": "v4.1.0",
		"Licence": "MIT",
		"URL": "https://zydis.re",
		"LicenceFile": "https://raw.githubusercontent.com/zyantific/zydis/v4.1.0/LICENSE"
	}
]
```

---

sure - makes sense :+1: